string mission_id
string ag_addr
---
bool done